hack/generator/pkg/astbuilder/builder.go

```diff
@@ -283,3 +256,36 @@ func QualifiedTypeName(pkg string, name string) *dst.SelectorExpr {
 		Sel: dst.NewIdent(name),
 	}
 }
+
+func Selector(expr dst.Expr, name string) *dst.SelectorExpr {
```

Comment:
missing comments and examples on this and a few methods below

Reply:
Added.